src/typings.ts

```diff
@@ -51,7 +51,7 @@ export interface Config {
   scripts: {
     "develop-course": string;
     "run-course": string;
-    test?: string;
+    test: string;
```

Comment:
`test` has to be optional, because it is in the config.

```suggestion
    test?: string;
```

Reply:
So, the issue was that you can't call `test` if test isn't in the config.

The fix I've come up with silently ignores that (probably not what you want), but I wasn't sure what you did want to do.